pkg/quarterdeck/auth.go

```diff
@@ -40,6 +41,9 @@ func (s *Server) Register(c *gin.Context) {
 		out *api.RegisterReply
 	)
 
+	// Initialize prometheus collectors (this function has a sync.Once so it's safe to call more than once)
+	metrics.Setup()
```

Comment:
Calling `metrics.Setup` in every handler is a bit of overkill -- I think it's probably sufficient to do it in in the `server.Routes` method (https://github.com/rotationalio/ensign/blob/main/pkg/quarterdeck/quarterdeck.go#L138) and if needed, in the `SetupSuite` method of the tests (https://github.com/rotationalio/ensign/blob/main/pkg/quarterdeck/quarterdeck_test.go#L35)

It doesn't hurt anything though so it shouldn't block merging the PR.